system/mp/mplog.cpp

```diff
@@ -24,7 +24,87 @@
 LogMsgChildReceiverThread * childReceiver;
 LogMsgParentReceiverThread * parentReceiver;
```

Comment:
Shouldn't be extending mplog.cpp

Our general logging system should be used (jlog).

mplog is effectively a jlog routing system. It is enabled via hooks and allows logs to be piped to other places , e.g. a central store.

For general logging use PROGLOG, or DBGLOG, or EXCLOG etc.